packages/react-native-reanimated/Common/cpp/reanimated/CSS/common/values/CSSColor.cpp

```diff
@@ -98,16 +106,19 @@ std::string CSSColor::toString() const {
     return "rgba(" + std::to_string(channels[0]) + "," +
         std::to_string(channels[1]) + "," + std::to_string(channels[2]) + "," +
         std::to_string(channels[3]) + ")";
-  } else {
-    return "transparent";
+  } else if (colorType == ColorType::CurrentColor) {
```

Comment:
Earlier branch returned, so else if is redundant here, there is a clang tidy check on this type things ;)
```suggestion
  if (colorType == ColorType::CurrentColor) {
```

Reply:
I know that there is a clang tidy check but I think that we don't have it properly configured yet. That's something we should do in the future if it hasn't been done yet.

I know that we have a [GA](https://github.com/software-mansion/react-native-reanimated/actions/workflows/lint-clang-tidy-nightly.yml) that runs clang tidy but it fails for a long time and noone cares about it